samples/dnn/action_recognition.py

```diff
@@ -0,0 +1,280 @@
+import os
+import copy
+import shutil
+import numpy as np
+import cv2 as cv
+import argparse
+
+parser = argparse.ArgumentParser(add_help=False)
```

Comment:
`-h` should print help so I think we need to drop it. Add also some description what sample do.